src/main/java/com/aws/greengrass/detector/uploader/ConnectivityUpdater.java

```diff
@@ -100,7 +107,8 @@ private UpdateConnectivityInfoResponse updateConnectivityInfo(List<ConnectivityI
     }
 
     //For Junit Testing
```

Comment:
TODO: These unit tests should be re-written to avoid needing these testing helper methods. Do this as part of a separate PR